### PR TITLE
Adds tooltips

### PR DESCRIPTION
**Pull request by kaw2k**

### CODE REVIEW

File: templates/digital-asset-template/frontend/components/ui/info.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { Tooltip, TooltipContent, TooltipTrigger } from "./tooltip";
+import { Info as InfoIcon } from "lucide-react";
+// import { cn } from "@/lib/utils";
```
Comment (0xaptosj) [inline templates/digital-asset-template/frontend/components/ui/info.tsx:4]: remove?

Comment (0xaptosj) [inline templates/digital-asset-template/frontend/components/ui/info.tsx:4]:
```suggestion
```

File: templates/digital-asset-template/frontend/pages/CreateCollection.tsx [hunk 2]
```diff
@@ -228,8 +228,13 @@ export function CreateCollection() {
               </Card>
             </div>
             <div className="mb-5 flex flex-col item-center space-y-4">
-              <Label>Royalty Percentage</Label>
+              <Label
+                tooltip="The percentage of trading fee that collection creator gets when an NFT is traded on marketplaces."
```
Comment (0xaptosj) [inline templates/digital-asset-template/frontend/pages/CreateCollection.tsx:232]:
```suggestion
                tooltip="The percentage of trading value that collection creator gets when an NFT is traded on marketplaces."
```

File: templates/digital-asset-template/frontend/pages/CreateCollection.tsx [hunk 3]
```diff
@@ -318,8 +333,13 @@ export function CreateCollection() {
             </div>
           </div>
           <div className="mb-5 flex flex-col item-center space-y-4">
-            <Label>Limit mint per address</Label>
+            <Label
+              htmlFor="mint-limit"
+              tooltip="How many NFTs an individual account is allowed to mint.">
```
Comment (0xaptosj) [inline templates/digital-asset-template/frontend/pages/CreateCollection.tsx:338]:
```suggestion
              tooltip="How many NFTs an individual address is allowed to mint.">
```

File: templates/digital-asset-template/frontend/pages/CreateCollection.tsx [hunk 4]
```diff
@@ -328,19 +348,29 @@ export function CreateCollection() {
             />
           </div>
           <div className="mb-5 flex flex-col item-center space-y-4">
-            <Label>Mint fee per NFT in APT (optional)</Label>
+            <Label
+              htmlFor="mint-fee"
+              tooltip="The fee the nft minter is paying the collection creator when they mint a NFT">
```
Comment (0xaptosj) [inline templates/digital-asset-template/frontend/pages/CreateCollection.tsx:353]:
```suggestion
              tooltip="The fee the nft minter is paying the collection creator when they mint an NFT">
```

File: templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx [hunk 5]
```diff
@@ -174,18 +184,28 @@ export function CreateFungibleAsset() {
               />
             </div>
             <div className="mb-5 flex flex-col item-center space-y-4">
-              <Label>Max Supply</Label>
+              <Label
+                tooltip="The total amount of the asset that can be created."
```
Comment (0xaptosj) [inline templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx:188]:
```suggestion
                tooltip="The total amount of the asset that can be minted."
```

File: templates/digital-asset-template/frontend/pages/CreateCollection.tsx (hunk 4 above)
Comment (kaw2k) [reply templates/digital-asset-template/frontend/pages/CreateCollection.tsx:353]: Not sure about the standards around this one. Is it common for `an` to be used with NFT?

File: templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx [hunk 6]
```diff
@@ -204,8 +229,11 @@ export function CreateFungibleAsset() {
               />
             </div>
             <div className="mb-5 flex flex-col item-center space-y-4">
-              <Label>Project URL</Label>
+              <Label tooltip="Your website address" htmlFor="project-url">
```
Comment (0xaptosj) [inline templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx:232]:
```suggestion
              <Label tooltip="Your website URL" htmlFor="project-url">
```

File: templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx [hunk 7]
```diff
@@ -214,17 +242,27 @@ export function CreateFungibleAsset() {
               />
             </div>
             <div className="mb-5 flex flex-col item-center space-y-4">
-              <Label>Mint fee per fungible asset in APT (Optional)</Label>
+              <Label
+                htmlFor="mint-fee"
+                tooltip="The fee cost for the minter to pay for every asset mint. For example, if a user mints 10 assets in a single transaction, they are charged 10x the mint fee.">
```
Comment (0xaptosj) [inline templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx:247]:
```suggestion
                tooltip="The fee cost for the minter to pay to mint one asset. For example, if a user mints 10 assets in a single transaction, they are charged 10x the mint fee.">
```

**Review by 0xmaayan — APPROVED** [review]

File: templates/digital-asset-template/frontend/pages/CreateCollection.tsx (hunk 4 above)
Comment (0xaptosj) [reply templates/digital-asset-template/frontend/pages/CreateCollection.tsx]: i think so, google said use a for non fungible token and an for NFT

**Pull request merged**